### ISSUE TITLE
Answer vocabulary

### ISSUE THREAD

**Issue by gkv91** [Cadene/murel.bootstrap.pytorch#19]
Hi,
How the answer vocabulary is selected (for VQAv2, VQA-CPv2 and TDIUC)?
Thanks

**Comment by Cadene** [comment]
Most common answers.
The number of answers is selected as an hyper parameter (grid search).

**Comment by gkv91** [comment]
Do you mean top 1000,2000,3000 like that?. Also, the accuracy scores correspond to the questions with these answers (those are in the vocabulary) or the entire dataset?
Thanks

**Comment by Cadene** [comment]
Yes :'D

On the validation set, it's only on the chosen subset.

On the testing set, it's for all of them.
